src/components/AddFoodForm.jsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from 'react';
+import { Divider, Input } from 'antd';
+
+const defaultFood = {
+  name: '',
+  image: '',
+  calories: 0,
```

Comment:
just be aware that these don't stay numbers, they become strings after your input is done. Luckily in the way that the data is used, it doesn't matter, but it might break if this is extended on.